services/app/src/pages/game/Game.jsx

```diff
@@ -80,7 +80,7 @@ function Room() {
 
   return (
     <div>
-      <Header username={localStorage.getItem('user')} roomUrl={roomUrl} />
+      <Header player={currentPlayer} username={localStorage.getItem('user')} roomUrl={roomUrl} />
```

Comment:
Do we need to pass `username` as a separate prop? Shouldn't `player` object include the username?